action.yml

```diff
@@ -77,6 +77,10 @@ inputs:
     description: "Whether to return the section markdown as output instead of writing to the file"
     required: false
     default: "false"
+  output_type:
```

Comment:
This input needs to get passed to the python script in order to be used.

Around line 121 of action.yml, it should pass this as a parameter to --output-type